ambry-server/src/integration-test/java/com.github.ambry.server/RouterServerTestFramework.java

```diff
@@ -352,6 +352,55 @@ void check() throws Exception {
     opChain.testFutures.add(testFuture);
   }
 
+  /**
+   * Submit a getBlob operation with incorrect accountId/ContainerId in blobId..
+   * @param opChain the {@link OperationChain} object that this operation is a part of.
+   */
+  private void startGetBlobAuthorizationFailTest(final OperationChain opChain) {
+    Callback<GetBlobResult> callback = new TestCallback<>(opChain, false);
+    BlobId originalId, fraudId = null;
+    try {
+      originalId = new BlobId(opChain.blobId, clusterMap);
+      fraudId = BlobId.craft(originalId, originalId.getVersion(), (short) (1), (short) 1);
```

Comment:
what if the account id, container id is 1? I think its probably better to do something like "current id + 1" so you are sure it is different

Reply:
Same with the delete